Agents/PropertyValueEstimationAgent/tests/conftest.py

```diff
@@ -0,0 +1,388 @@
+        print(f'PostgreSQL endpoint: {service_url}')
+
+        # This will run only once per entire test session
+        # It ensures that the requested PostgreSQL Docker service is ready to accept queries
+        service_available = False
+        while not service_available:
+            try:
+                conn = pg.connect(host=hostname, port=service.host_port,
+                                  user=DB_USER, password=DB_PASSWORD,
+                                  database=DATABASE)
+                if conn.status == pg.extensions.STATUS_READY:
+                    service_available = True
+            except Exception:
+                time.sleep(3)
+
+        print('Connected to PostgreSQL.')
+        return service_url
+    return _get_service_url
+
+
+@pytest.fixture(scope="session")
+def initialise_clients(get_blazegraph_service_url, get_postgres_service_url):
+    # Retrieve "user-facing" endpoints for all clients/services relevant for testing
+    # (i.e. invoked during testing from outside the Docker stack)
+    # --> those shall be `localhost:...` even when agent is running as Docker container
+    
+    # Retrieve endpoint for triple store
+    sparql_endpoint = get_blazegraph_service_url(KG_SERVICE, url_route=KG_ROUTE)
+
+    # Retrieve endpoint for postgres
+    rdb_url = get_postgres_service_url(RDB_SERVICE, url_route=RDB_ROUTE)
+
+    # Create SparqlClient for testing
+    sparql_client = KGClient(sparql_endpoint, sparql_endpoint)
+
+    # Create DerivationClient for creating derivation instances
+    derivation_client = PyDerivationClient(
+        derivation_instance_base_url=DERIVATION_INSTANCE_BASE_URL,
+        query_endpoint=sparql_client.query_endpoint,
+        update_endpoint=sparql_client.update_endpoint,
+    )
+
+    yield sparql_client, derivation_client, rdb_url
+
+    # Clear logger at the end of the test
+    clear_loggers()
+
+
+# ----------------------------------------------------------------------------------
+# Module-scoped test fixtures
+# (i.e. the fixture is destroyed during teardown of the last test in the module)
+# ----------------------------------------------------------------------------------
+
+@pytest.fixture(scope="module")
+def create_example_agent():
+    def _create_example_agent(
+        register_agent:bool=False,
+        alter_agent_iri:bool=False,
+    ):
+        agent_config = config_derivation_agent(AGENT_ENV)
+        agent = PropertyValueEstimationAgent(
+            register_agent=agent_config.REGISTER_AGENT if not register_agent else register_agent,
+            agent_iri=agent_config.ONTOAGENT_SERVICE_IRI if not alter_agent_iri else \
+                      agent_config.ONTOAGENT_SERVICE_IRI + '_altered',
+            time_interval=agent_config.DERIVATION_PERIODIC_TIMESCALE,
+            derivation_instance_base_url=agent_config.DERIVATION_INSTANCE_BASE_URL,
+            # NOTE Ensure SPARQL endpoints contain'docker.host.internal' for dockerised agent tests
```

Comment:
Please change to `host.docker.internal` in the comment, instead of `docker.host.internal`.

Reply:
Addressed by [70bf83d](https://github.com/cambridge-cares/TheWorldAvatar/pull/454/commits/70bf83da2a3c88d28f664d112439d09e18003cba)